jobflow.ipynb

```diff
@@ -908,11 +914,20 @@
    "cell_type": "markdown",
    "source": [
     "## Publication of the workflow\n",
-    "The `jobflow` infrastructure does not provide a dedicated platform for publishing a workflow currently. However, workflows related to computational materials science have been collected in the package `atomate2`. In addition, users can build their own package by relying on jobflow and share it as a new Python-based program. Additional packages in materials science using `jobflow` exist."
+    "The `jobflow` infrastructure does not provide a dedicated platform for publishing a workflow currently. However, workflows related to computational materials science have been collected in the package `atomate2`. In addition, users can build their own package by relying on `jobflow` and share it as a new Python-based program. There are also additional materials science packages that rely on `jobflow`."
```

Comment:
@JaGeo , do you know an example outside of the MP world?

Reply:
Check out the publication on jobflow plus QuAcc.

Reply:
Although QuAcc feels like half part of the MP 😃